api/test/data/import/.gitignore

```diff
@@ -0,0 +1 @@
+!api
```

Comment:
Why :thinking: ?

Reply:
Because otherwise I could not easily add to git the `api` directory of the sample packages in the `test/data` directory.
It already happened in the previous PR but I realized it only today whan I saw the CI tests failing because some files were not committed to git.
For reference:  https://github.com/keptn/keptn/blob/master/api/.gitignore#L10

Reply:
oh because we ignore the go binary in the main folder :D Thanks!